demos/ws-bootable-jar-example/jaxws/src/test/java/org/jboss/jaxws/SoapWildflyBootableOpenShiftJarApplication.java

```diff
@@ -0,0 +1,105 @@
+/*
+ * JBoss, Home of Professional Open Source.
+ * Copyright 2023, Red Hat Middleware LLC, and individual contributors
+ * as indicated by the @author tags. See the copyright.txt file in the
+ * distribution for a full listing of individual contributors.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ */
+package org.jboss.jaxws;
+
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+import org.apache.maven.settings.building.SettingsBuildingException;
+import org.eclipse.aether.resolution.ArtifactResolutionException;
+import org.jboss.intersmash.deployments.util.maven.ArtifactProvider;
+import org.jboss.intersmash.tools.application.openshift.BootableJarOpenShiftApplication;
+import org.jboss.intersmash.tools.application.openshift.input.BinarySource;
+
+import cz.xtf.builder.builders.SecretBuilder;
+import cz.xtf.builder.builders.secret.SecretType;
+import io.fabric8.kubernetes.api.model.EnvVar;
+import io.fabric8.kubernetes.api.model.EnvVarBuilder;
+import io.fabric8.kubernetes.api.model.Secret;
+
+public class SoapWildflyBootableOpenShiftJarApplication implements BootableJarOpenShiftApplication {
+	// private String GROUPID = IntersmashSharedDeploymentsProperties.groupID();
```

Comment:
Could you please remove this comment (and the similar one below) and create an issue for improving this by implementing something similar to https://github.com/Intersmash/intersmash/blob/main/deployments/intersmash-deployments-provider/src/main/java/org/jboss/intersmash/deployments/IntersmashSharedDeploymentsProperties.java ? I don't want to block the PR for this, but we need to have it tracked since could led to the wrong artifact being picked up in future.

Reply:
done
https://github.com/Intersmash/intersmash/issues/91